src/components/MainActionButton.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface mainActionButtonInterface {
+	text: string;
+	onClick: () => void;
+}
+
+const MainButton = styled.button`
+	background-color: #4961e1;
+	color: white;
+	font-size: 1em;
+	border-radius: 6px;
+	padding: 0.5rem 2rem;
```

Comment:
Will you also remove the border, add  add a hover effect to match the figma style guide? We can also set the width to 100% so that the button increases to be the same size as its container, this would be good to be able to change the button size as needed. One way to change these styles is to add the following to your `styled.button` attributes:
`width: 100%;
border: none;
	&:hover {
		background-color: #354bc4;
	}`